CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## v0.x.x Unreleased
+
+### New features
+
+### Maintenance and fixes
+
+### Deprecation
+
+### Documentation
+
```

Comment:
Very small nit about dates in versions: at line 13 we have `2025 Jul 9` and at line 30 we have `2025 Mar 06`. Could we always use the same number of digits?